image_generation/lcm_dreamshaper_v7/cpp/README.md

```diff
@@ -0,0 +1,103 @@
+# OpenVINO Latent Consistency Model C++ pipeline
+The pure C++ text-to-image pipeline, driven by the OpenVINO native API for SD v1.5 Latent Consistency Model with LCM Scheduler. It includes advanced features like LoRA integration with safetensors and [OpenVINO extension for tokenizers](https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/custom_operations/user_ie_extensions/tokenizer/python/README.md). This demo has been tested for Linux platform only.
+
+> [!NOTE]
+>This tutorial assumes that the current working directory is `<openvino.genai repo>/image_generation/lcm_dreamshaper_v7/cpp/` and all paths are relative to this folder.
+
+## Step 1: Prepare build environment
+
+C++ Packages:
+* [CMake](https://cmake.org/download/): Cross-platform build tool
+* [OpenVINO](https://docs.openvino.ai/2023.2/openvino_docs_install_guides_overview.html): Model inference
+* Eigen3: Lora enabling
```

Comment:
```suggestion
* Eigen3
```
`add_subdirectory(../../common/diffusers _deps/diffusers)` is unconditional, which requires Eigen3. That makes Eigen3 required not only for lora